randovania/game_description/resources/pickup_index.py

```diff
@@ -30,3 +30,68 @@ def __eq__(self, other):
     @property
     def index(self) -> int:
         return self._index
+
+    @property
+    def is_major_location(self) -> bool:
+        return self._index in _MAJOR_LOCATIONS
+
+_MAJOR_LOCATIONS = [
```

Comment:
This should come from the data somehow.

A new field on PickupNodes that describes if they're major/minor? Better yet would be the actual original pickup there, but that can come later.